Publisher/nl/database-import.md

```diff
@@ -1,116 +1,67 @@
 # Importeren van profielgegevens
-Er zijn verschillende manieren om nieuwe gegevens in te voeren of te importeren.
-Zo kun je profielen niet alleen met de hand toevoegen, maar kun je ze ook
-aanmaken of bewerken met de API, of importeren vanuit een CSV- of TXT-bestand
-(dit zijn bestanden die je onder meer in spreadsheetprogramma's kunt maken).
+Er zijn verschillende manieren waarop je nieuwe profielen binnen je database kunt toevoegen of wijzigen. Zo kun je profielen met de hand, via de [API](./apis), via [webformulieren](./webforms) of door middel van een import toevoegen, wijzigen of verwijderen. 
 
-Het importeren van CSV- of TXT-bestanden is krachtig en kan ook worden gebruikt
-voor gelaagde databases met collecties en subprofielen. Ook kun je periodieke
-imports maken die automatisch worden herhaald.
+De importfunctie in Copernica ondersteunt CSV- en TXT-bestanden. Je kunt deze bestandsformaten exporteren vanuit tekstverwerkingsprogramma's als Microsoft Word of spreadsheetprogramma’s als Excel.
 
 ## Importbestand voorbereiden
-Het bestand dat je wilt importeren moet aan een specifiek formaat voldoen.
-Het moet een tab-, komma of puntkommagescheiden bestand zijn, waarvan de bovenste regel
-de kolommen (de veldnamen) bevat die je gaat importeren. Het bestand moet een
-UTF-8 encoding hebben en wij raden het aan om quotes om veldwaardes te plaatsen.
-Hieronder geven we een voorbeeld van hoe dit bestand eruit kan zien.
+Naast het feit dat een importbestand uit CSV- of TXT-bestanden moet bestaan dienen de bijbehorende kolommen ook door een tab, komma of puntkomma gescheiden te zijn. Ook dient de bovenste regel de veldnamen te bevatten die je wilt importeren en moet het bestand voorzien zijn van UTF-8 encoding.
+
+Wanneer je gebruik maakt van scheiding door middel van een komma of puntkomma raden we aan om veldwaardes van enkele (‘) of dubbele (“) aanhalingstekens te voorzien. Zo worden komma’s of puntkomma’s niet als scheidingsteken beschouwd. Onderstaand zie je hiervan een voorbeeld:
```

Comment:
De getoonde voorbeeldtekens zijn geen enkele of dubbele ascii-quote, maar een ingewikkeld schuingedrukt quote-teken